src/main/java/eu/de4a/connector/mock/exampledata/DataOwner.java

```diff
@@ -13,7 +13,8 @@
     UL_PT("iso6523-actorid-upis::9999:PT990000101", "Portuguese IST, University of Lisbon", "PT", Pilot.T41),
     JSI_SI("iso6523-actorid-upis::9999:SI000000018", "(JSI) Institut Jozef Stefan", "SI", Pilot.T41),
     MIZS_SI("iso6523-actorid-upis::9999:SI000000016", "(MIZS) Ministrstvo za Izobrazevanje, Znanost in Sport", "SI", Pilot.T41),
-    UJI_ES("iso6523-actorid-upis::9999:ESQ6250003H", "(UJI) Universitat Jaume I de Castellón", "ES", Pilot.T41);
+    UJI_ES("iso6523-actorid-upis::9999:ESQ6250003H", "(UJI) Universitat Jaume I de Castellón", "ES", Pilot.T41),
+    SGAD_ES("iso6523-actorid-upis::9999:ESS2833002E", "(MPTFP-SGAD) Secretaría General de Administración Digital", "ES", Pilot.T41);
```

Comment:
I vaguely remember @phax commenting somewhere else that identifiers should be lowercase

Reply:
so ess2833002e

Reply:
The values are copy-pasted from the dataset matrix. The matching by the mock ignores case, so I think this is fine.